modules/bloxone_anycast/variables.tf

```diff
@@ -0,0 +1,64 @@
+variable "hosts" {
```

Comment:
nested fields that are optional should be marked as `optional(string/list...)`

Reply:
done